geometry_msgs/Pose[] target_pose_vec
float64 delta_rotation_angle
float64 delta_move_dist
---
geometry_msgs/Pose[] nav_pose_vec
